### ISSUE TITLE
In English Wiktionary all collapsible elements are collapsed and can't be accessed

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#2073]
**[<img src="https://avatars.githubusercontent.com/u/58372007?s=17&v=4" width="17" height="17"> @bicolino34](https://github.com/bicolino34)** cloned issue [kiwix/kiwix-desktop#1135](https://github.com/kiwix/kiwix-desktop/issues/1135) on 2024-06-24: 

> Kiwix version 2.3.1 (installed through flatpak)
> 
> OS Linux Mint 21
> 
> 
> 
> I have downloaded English Wiktionary (text-only) file for offline use in Kiwix, but the translation table is collapsed by default and there is no way to expand it. No matter how many times I click it. It also affects declension data because in English Wiktionary it is shown in collapsible element.
> 
> ![зображення](https://github.com/kiwix/kiwix-desktop/assets/58372007/7501b615-410f-43ea-8121-432598eae7d1)
> 
> ![зображення](https://github.com/kiwix/kiwix-desktop/assets/58372007/33b89e04-890e-4091-a20e-9f861c7447fd)
> 
> 
> 
> It is probably some problem with parsing, since Ukrainian Wiktionary seems to have this elements expanded with all the content shown.
> 
> ![зображення](https://github.com/kiwix/kiwix-desktop/assets/58372007/a62974ab-bd21-43bf-84a0-e430201f2c1a)
> 
>

**Comment by kelson42** [comment]
**[<img src="https://avatars.githubusercontent.com/u/1029718?s=17&v=4" width="17" height="17"> @kelson42](https://github.com/kelson42)** [commented](https://github.com/kiwix/kiwix-desktop/issues/1135#issuecomment-2191445692) on 2024-06-26: 

> This issue should be moved to `openzim/mwoffliner` but this is a known issue (which might already been fixed in `dev`).

**Comment by kelson42** [comment]
[@bicolino34](https://github.com/bicolino34) I believe we are going to remove this whole collapsing feature, so this tight might actually become invalidt with release of `1.14`. To be confirmed.

**Comment by Jaifroid** [comment]
Duplicate of #1033 (or a more general case of that longstanding issue).

**Comment by maksbotan** [comment]
Hi. This bug is still present in latest (as far as I can tell) English Wiktionary zim both on Kiwix Android and online: https://library.kiwix.org/viewer#wiktionary_en_all_maxi_2024-05/A/visit (try clicking on any table).

If it's really fixed in next release of mwoffliner, is there a timeline when this fix will hit Kiwix?

**Comment by ewtoombs** [comment]
I wrote a workaround for Greasemonkey:

https://greasyfork.org/en/scripts/517632-kiwix-wiktionary-toggle-navcontent

You'll probably have to modify the @include directive.

**Comment by Jaifroid** [comment]
@ewtoombs Thanks for the ideas in that script. We need to succeed in producing a scrape of Wiktionary with the dev-1.14 build (see #2098), where hopefully some of these problems may be fixed (or else it will become clearer what will be needed to fix them). @audiodude you might be interested in this.

**Comment by kelson42** [comment]
Closing as we foresee ActionParse only MWofffliner 2.0.0

**Comment by maksbotan** [comment]
@kel hi!

Can you give an estimate when fixed ZIMs of Wiktionary will be available for download? Latest one in Kiwix for android is from 2024...

**Comment by kelson42** [comment]
@maksbotan Indeed, I have restarted one scrape manually, see https://farm.openzim.org/pipeline/11f807e8-aacb-4eea-8a73-f940261d34d0. Goalis definitly to have one in July.

**Comment by maksbotan** [comment]
@kelson42 hi! Looks like the scrape from your link has failed... Can you please look into that?

**Comment by kelson42** [comment]
@maksbotan Indeed, we are working to fix the bug ASAP to telaunch the scrape

**Comment by maksbotan** [comment]
Thank you! Can you please ping when the new scrape is started?

**Comment by kelson42** [comment]
@maksbotan As soon as https://github.com/openzim/mwoffliner/issues/2408 is fixed, I will restart a scrape
